<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HisData</class>
 <widget class="QWidget" name="HisData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbTPC">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>细菌总数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTolColi">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>总大肠菌群</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbFecColi">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>耐热大肠菌群</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>800</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择所需查看的历史数据！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>410</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>641</width>
     <height>280</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pbTemp2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>温区2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEColi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>大肠埃希氏菌</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTemp1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>温区1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTemp3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>温区3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTemp4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>温区4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTemp5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>温区5</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_Title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>历史数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDataOut">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>数据导出</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pbTPC</zorder>
  <zorder>pbTolColi</zorder>
  <zorder>pbFecColi</zorder>
  <zorder>line_2</zorder>
  <zorder>label</zorder>
  <zorder>pbExit</zorder>
  <zorder>lb_Title</zorder>
  <zorder>pbDataOut</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
